src/context/ConstantContext.tsx

```diff
@@ -2,12 +2,14 @@ import { LabelStringSet } from '@sendbird/uikit-react/ui/Label';
 import { createContext, useContext, useMemo } from 'react';
 
 import { type Constant, DEFAULT_CONSTANT } from '../const';
+import { SendbirdChatAICallbacks } from '../interfaces';
 
 const initialState = DEFAULT_CONSTANT;
 
 interface ConstantContextProps extends Constant {
   applicationId: string | null;
   botId: string | null;
+  callbacks?: SendbirdChatAICallbacks;
```

Comment:
For DX, it would be good to align the naming conventions of **function calls** and **callbacks** interfaces to some extent.

Reply:
or, please use a specific name for onViewDetailClick that is closely related to FunctionCall.

Reply:
I choose name `callbacks` intentionally to make it as general as possible. It should contain any callbacks for future not just for this function callback case. Currently it only has `onViewDetailClick` but it can include many more other cases in the future.

current
```
export interface SendbirdChatAICallbacks {
  onViewDetailClick?: (data: ViewDetailData) => void;
}
```
I was not sure of name `onViewDetailClick` is good or not. Do you have suggestion? @bang9

Reply:
While handling only data related to `FunctionCall`, the naming of `ViewDetail` seems too general.
Moreover, there is no guarantee that all data, including `FunctionCall`, added in the future will be provided in the same format.
Therefore, it would be good to make the naming more specific to the current use case. (If there are common fields in the format of data added in the future, we can create a separate interface for that part.)

Reply:
Can you provide an example for the naming?

Reply:
callbacks has onXxxClick convention so i think name looks ok but ViewDetailData -> FunctionCallData looks better

Reply:
updated
```
export interface SendbirdChatAICallbacks {
  onViewDetailClick?: (data: FunctionCallData) => void;
}
```